Improve sentence structure and grammar. (#423)
lonnieezell/myth-auth: README.md

load the helper before using these functions: `helper('auth');`

**Hint**: Add `'auth'` to any controller's `$helper` property to have it loaded automatically,

> or the same in **app/Controllers/BaseController.php** to have it globally available. The
> `'auth'` filters and pre-loads the helper so that it is available on any filtered routes.

Reviewer: I think that the confusion is because "filters" is actually a noun in this case. How about something like this to make that more clear:
```suggestion
or the same in **app/Controllers/BaseController.php** to have it globally available.
All of this library's Filters preload the helper so that it is available on filtered routes.
```

Author: Sounds good.

> `'auth'` filters and/or pre-loads the helper so that it is available on any filtered routes.

Reviewer: I think you made this one worse.

Author: " the auth filters all pre-load the helper so it is available on any filtered routes." ...this sentence does not make sense to me anyway, if anything I am confused by it.

Author: @MGatner I have made changes based on how I understand the flow. Hope it is correct this time and makes grammatical sense to you and all.

Reviewer 2: When is specifies `filters` here it's a noun, not a verb. It means the Controller Filters, so maybe that should be uppercase to avoid confusion?
